src/hooks/usePositioning.tsx

```diff
@@ -108,7 +142,8 @@ export const usePositioning = (
                         resolvedAnchorEl as HTMLDivElement,
                         {...anchorElOrigin, horizontal: 'left'},
                         {...transformElOrigin, horizontal: 'right'},
-                        anchorPosition
+                        anchorPosition,
```

Comment:
I think it would be better to pass both `resolvedAnchorEl` (the original one) and the `closeRelativeAncestor` as separate parameters. `resolvedAnchorEl` is used to calculate `anchorElRectangle`, which give the size of the anchor - used for calculating center/top/bottom and the tranformElOrigin. If we replace resolvedAnchorEl with closeRelativeAncestor, the rectangle size may be completely different.

Reply:
My understanding is that when the menu position is set to `absolute`, then we want to check ancestors for the closest positioned element, anchor against that, and use that to calculate the rectangle. Also, when position is `absolute`, aren't we saying that the user might not need to pass in an element for the `anchorEl` prop?